srcs/juloo.keyboard2/Autocapitalisation.java

```diff
@@ -149,7 +149,7 @@ void callback(boolean might_disable)
     _should_disable_shift = might_disable;
     // The callback must be delayed because [getCursorCapsMode] would sometimes
     // be called before the editor finished handling the previous event.
-    _handler.postDelayed(delayed_callback, 1);
+    _handler.postDelayed(delayed_callback, 50);
```

Comment:
What bug is this fixing ? This is an important information that I would like to have as a comment in the code.